rust-manifest/src/config.rs

```diff
@@ -9,19 +10,7 @@ pub const DEFAULT_CONFIG_VERSION: &'static str = "1";
 #[derive(Clone, Debug)]
 pub struct Config {
     pub config_version: String,
-    pub remote: Option<ConfigRemote>,
-    pub install: ConfigInstall,
```

Comment:
Are we getting rid of these kind of configuration options for now? I think the ability to specify a custom remote is potentially very useful.